static/js/stream_edit.js

```diff
@@ -204,13 +204,17 @@ function show_subscription_settings(sub_row) {
             return typeahead_helper.render_person(item);
         },
         matcher: function (item) {
+            var stream_id = this.$element.closest(".subscription_settings").attr("data-stream-id");
```

Comment:
I think we already have `stream_id` in the outer scope, so we don't need this line.